### DISCUSSION TITLE
Extend colors/theme to eliminate blue stock UI entirely.

### DISCUSSION THREAD

**Discussion by ssshake** [Jyers/Marlin#543] [Ideas]
I really like the option to change colors in this firmware. I like it a lot. It's not just a personalization preference, it's a major usability enhancement. 

Blue looks horrible on the ender 3 v2 display, it's bad at almost every viewing angle. It's comical that of all the colors they could have picked, they choose the least resilient one and used that for a nearly monochrome theme of all blue and grey. 

I also love that you added an outline color to the main screen selection icons. This feeds into my point exactly. You added that feature because the default highlight color of some shade of blue/grey was quite impossible to see at angles you'd normally find yourself at around a printer. 

The screen with its default color scheme is hard to see. 

I've noticed that green, white and magenta all display quite well even at bad angles. I have customized my UI to only use these colors, however all of the icons and the header/title row on the screen are the stock blue.

I would be very happy if this was customizable. Is it at all possible?

**Comment by AngleV** [comment]
@ssshake 

Hello

The header/title row color setting is in my latest pull request. 

https://github.com/Jyers/Marlin/pull/528

We 'll just wait for jyers to merge the commit.
To change the stock icons you have to update the LCD firmware.
You can use this file if  you prefer.

https://drive.google.com/file/d/1a2x-StrkKZxCPnASHgdXZkMtvmOiHiXO/view

Do a search in the repo for the procedure of how to update the LCD.

**Reply by ssshake** [reply]
@AngleV aweseome!

**Reply by ssshake** [reply]
What tool would I use to view that ico file? I tried a few difference tools but they didn't know how to read it.

**Reply by AngleV** [reply]
Read this

https://github.com/Jyers/Marlin/tree/Ender_3_V2_Extensible_UI/Display%20firmware/dwin-ico-tools
